### PR TITLE
Add test file fixtures to ease up tests and new words and rules

### PR DESCRIPTION
**Pull request by lucasmichot**

### CODE REVIEW

File: tests/Doctrine/Tests/Fixtures/InflectorWords.json [hunk 1]
```diff
@@ -0,0 +1,266 @@
+[
```
Comment (Ocramius) [inline tests/Doctrine/Tests/Fixtures/InflectorWords.json]: Please revert to the good 'ol PHP array

Comment (GrahamCampbell) [inline tests/Doctrine/Tests/Fixtures/InflectorWords.json]: this is a json file though?

**Comment by Ocramius** [comment]
Closing as invalid - can re-apply if the json is removed and reverted back to a PHP data provider.

File: tests/Doctrine/Tests/Fixtures/InflectorWords.json (hunk 1 above)
Comment (GrahamCampbell) [inline tests/Doctrine/Tests/Fixtures/InflectorWords.json]: oh, right, i see what you mean now ;)

**Pull request closed without merging**